src/main/java/org/broadinstitute/consent/http/db/DatasetAssociationDAO.java

```diff
@@ -35,7 +35,7 @@ public interface DatasetAssociationDAO extends Transactional<DatasetAssociationD
     @SqlQuery("select exists (select * from dataset_user_association where datasetId = :datasetId )")
     Boolean exist(@Bind("datasetId") Integer datasetId);
 
-    @SqlQuery("select ds.dacuserId from dataset_user_association ds where ds.dataset_id = :datasetId")
+    @SqlQuery("select ds.dacuserId from dataset_user_association ds where ds.datasetId = :datasetId")
```

Comment:
We don't have a DAO test for this, please add one.

Reply:
As a matter of fact, we don't have any DAO tests for this class :-( Can we add that here to ensure completeness?

Reply:
Added DatasetAssociationDAOTest and tests for all these queries,